src/thread.h

```diff
@@ -14,6 +14,8 @@
 #include "crashlog.h"
 #include <system_error>
 #include <thread>
+#include <mutex>
+#include <condition_variable>
```

Comment:
`std::lock_guard` is in `<mutex>` and so is `std::mutex`. Where do we need `<condition_variable>`?